common/build.gradle

```diff
@@ -0,0 +1,43 @@
+
+apply plugin: 'java-library'
+apply plugin: 'java-test-fixtures'
+
+jar {
+  archiveBaseName = 'web3signer-common'
+  manifest {
+    attributes(
+      'Specification-Title': archiveBaseName,
+      'Specification-Version': project.version,
+      'Implementation-Title': archiveBaseName,
+      'Implementation-Version': calculateVersion()
+      )
+  }
+}
+
+dependencies {
+
+  implementation 'com.google.guava:guava'
```

Comment:
There are some dependencies here that I don't think we need for the Jackson serialisers guava, commons-io, commons-lang, web3j, jackson-cbor, jackson-yaml but perhaps these are transitive dependencies we need.

Reply:
cleaned up.